contracts/LiquidityConversionRates.sol

```diff
@@ -1,57 +1,249 @@
-pragma solidity ^0.4.18;
+pragma solidity 0.4.18;
 
-import "./ConversionRates.sol";
+import "./ConversionRatesInterface.sol";
 import "./LiquidityFormula.sol";
+import "./Withdrawable.sol";
+import "./Utils.sol";
 
-contract LiquidityConversionRates is ConversionRates, LiquidityFormula {
-    uint public r;
-    uint public Pmin;
-    uint public precision;
-    uint public numPrecisionBits;
 
+contract LiquidityConversionRates is ConversionRatesInterface, LiquidityFormula, Withdrawable, Utils {
+    ERC20 public token;
+    uint public rInFp;
+    uint public pMinInFp;
+    uint public formulaPrecision;
+    uint public numFpBits;
+    uint public maxCapBuyInFp;
```

Comment:
empty lines between some of the variables would help...
e.g., `maxCapBuyInFp` and `maxCapSellInFp` can be together.